src/xebia_email_signature/templates/new_form.html

```diff
@@ -0,0 +1,517 @@
+                        value="event"
+                      >
+                        Find me in the event
+                      </option>
+                      <option
+                        data-custom-properties="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABYAAAATCAYAAACUef2IAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAIESURBVHgBtVS9UsJAED4CQ2t8AmMPTErL0NmJHTDMGEqr4BMgTyA8AXGGv47wBEJnZ2B4gFDaxYYBhh+/xbsYMYFYuDM3u7f37Xe7ez+M/ZPEgpy6riur1eoFpsKOi5NMJrOmaTqHC4kgNEh1qHGn07lkR6RYLPaXy2UOZj2QGIAmlCacsVhM3u12FjshwHxIklRFvOGLHbbb7bLIWIOjgWHTZLPZ6LBP8RLJ2Xa7HcTjcZPHqVCGlzEH2a1Wa8grUKByEYhlqIGIK5VKDBvt1yQOcKFkEYDyXPgu2GmRgXX8c8Q5HjF2GWNoYnW9Xtvon3KKFRg1kUjYvvkNhuMR8x7dCUCv13Oox2iJGkaKsjUoB1fN9RFrqODZI+Y9cguFQkWA6FDYkT5jndaGvo0MH9cX8d6QpDKyrIoseRUGC5cbkR0elIqN6hhZsRgXxmQycVKp1BLkzXQ6/Y67aGUymWv4zqfT6Sv72QYDZV8B84Aqc7D7cN93u92hwPy6rMhYh6rSRUfACPYTRhav0BbZ4WW+waQK6VxU6FvRglBiknw+rwD8SIF0FbEBHUyNHgRGBbbM/Y35fF63LMs95Dj6vGgD9FGjfrLvgzQxRiC0ggj/LPTh8D8lkkhRgfThQM2i4kNbwf/kqs8lHov30vAX14L+YpLA/5hksVjQ3fZnOAvChMknpEbxAnSaXUQAAAAASUVORK5CYII="
+                        value="award"
+                      >
+                        Special / Award
+                      </option>
+                    </select>
+                    <div id="cta-icon-error" class="error"></div>
+                  </div>
+                  <div class="form-group w-100 mb-0">
+                    <div class="relative flex items-center">
+                      <img
+                        class="input-icon-img"
+                        src="/static/icon-link.svg"
+                        alt="Icon Watch"
+                      />
+                      <input
+                        class="input-icon"
+                        type="text"
+                        id="cta-0-link"
+                        name="cta[0][link]"
+                        value="https://www.xebia.com/news"
+                      />
+                    </div>
+                    <div id="cta-link-error" class="error"></div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="form-group">
+            <button
+              class="btn btn-md btn-dawn w-100 text-left font-semibold js-form-clone-btn"
+              data-clone-target=".js-form-cta"
+            >
+              <img src="static/icon-btn-add.svg" alt="Icon add" />
+              Add another Call to Action
+            </button>
+          </div>
+
+          <div class="form-bottom">
+            <a
+              class="btn btn-outline btn-small"
+              href="static/design.pdf"
```

Comment:
This design.pdf is referring to the old design

Reply:
Yeah, the new file is in progress. For now we'll hide this button.